js/APY.js

```diff
@@ -1,5 +1,11 @@
   var publicSpreadsheetUrl = 'https://docs.google.com/spreadsheets/d/1ut6PZe7GcGd2TBqpVPQcERh2nkhfMNx-TKFj0l8paFU/edit?usp=sharing';
-  
+  var osmosisStatus = 'https://api-osmosis.cosmostation.io/v1/status'
```

Comment:
need to be tested before merge

Reply:
The APR doesn't show for me locally. I'll try to find out what's wrong.